### DISCUSSION TITLE
Individual /eco perms!

### DISCUSSION THREAD

**Discussion by ryankayz7** [EssentialsX/Essentials#4050] [Ideas and Feature Suggestions] [answered]
So I was looking to add /eco take or /eco reset perms only for my mods however I don't think this is possible at the moment. I don't want to allow them access to /eco give or set as they are super easy to abuse. Is there a chance permissions for eco can be individualised kind of like for example "eco.give" or "eco.take". Another suggestion I have alongside this is to have a perm that means you can't do /eco reset ** but you can do /eco reset for an individual player. For example "eco.reset.all". I often find this useful when an xrayer's balance is needed to be reset. However I can't give mods these perms without the potential of them abusing it. It would be great if these permissions could be added or similar permissions but if not I understand!

**Comment by JRoy** [comment, marked as answer]
Cannot be done without breaking compatibility

**Reply by ryankayz7** [reply]
All good! One of our server devs just made a balancereset command as a work around (:
